src-tauri/src/main.rs

```diff
@@ -51,9 +51,8 @@ fn inject_specs(dll: String) -> Result<(), String> {
             Ok(_) => {}
             Err(err) => {
                 warn!("Failed to inject dll: {}", err);
-                return;
```

Comment:
not sure i like this change. if i ever add code under the inject_dll this is now a bug.

Reply:
Good point. I reworked this to use a (throwaway) `Result` for short-circuiting. I first wrote it using `Option` to get rid of the error, but coercing into `anyhow::Error` gets rid of `Option.ok()` calls.

Sadly, `inspect_err` is on nightly, so we pass it through manually. An alternative would be using `()` as the error type, but that feels like a puzzle waiting to happen.

Reply:
Also, the complier fails to infer the error type. I opted for explicitly typing the closure, but there's a few places it could go.

Reply:
LMK if you'd prefer the old `match` approach. Slapping `#![allow(clippy::whatever)]` is always an option